.github/workflows/pr.yaml

```diff
@@ -75,6 +76,13 @@ jobs:
     with:
       build_type: pull-request
       script: "ci/test_python_singlegpu.sh"
+  conda-python-tests-integration:
```

Comment:
Naming nitpick: can we call it "...-cudf-pandas-integration"? Something to hint at that the integration that is being tested is with `cudf.pandas`.